### PR TITLE
/v2/inv-playbook/task/form/submit args fix

### PR DESCRIPTION
**Pull request by Itay4**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/33333

## Description
- updated the swagger with the fix for the communication task api args
- dropped the support for py2

## Must have
- [x] Changelog entry

### CODE REVIEW

File: server_api_swagger.json [hunk 1]
```diff
@@ -181,13 +181,6 @@
             "name": "file",
             "in": "formData",
             "required": true
-          },
-          {
-            "type": "string",
-            "description": "associated typeID for the layout",
-            "name": "classifierId",
```
Comment (glicht) [inline server_api_swagger.json:188]: why was this removed?

Comment (Itay4) [reply server_api_swagger.json:188]: removed in https://github.com/demisto/server/commit/9bbf53f54c54abab4a8895f01de69e5de706c51d
@aviadl can you suggest here please?

**Comment by Itay4** [comment]
@glicht ive kept here only the fix for the bug

**Comment by yaakovi** [comment]
please update the PR description

**Comment by Itay4** [comment]
> please update the PR description

done

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,10 @@
 [PyPI History][1]
 
 [1]: https://pypi.org/project/demisto-py/#history
+## 2.0.24
```
Comment (glicht) [inline CHANGELOG.md:6]: Let's bump to 3.0.0 as we are dropping py 2.7.

**Review by glicht — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (Itay4) [reply CHANGELOG.md]: done

**Review by glicht — APPROVED** [review]
Looks good
